Translation-Chinese.md

```diff
@@ -5,7 +5,8 @@
 [Français](./Translation-French.md) ·
 [Русский](./Translation-Russian.md) ·
 [Deutsch](./Translation-German.md) ·
-[日本語](./Translation-Japanese.md)
+[日本語](./Translation-Japanese.md) .
```

Comment:
This should be a [middle dot](https://en.wiktionary.org/wiki/·), not a regular dot character. Copy-paste it from the line above. Please fix it in other files as well.